packages/components/spacings/inline/README.md

```diff
@@ -1,10 +1,12 @@
 # Inline
 
+#### Description
```

Comment:
Use `##` instead of `####`. See other packages, for example https://raw.githubusercontent.com/commercetools/ui-kit/96fcc4fd1e8ea90b14c93270cc6c0f2ab1f16e80/packages/components/avatar/README.md

Reply:
Some other has `####` instead of `##` i.e. https://raw.githubusercontent.com/commercetools/ui-kit/master/packages/components/inputs/toggle-input/README.md
https://raw.githubusercontent.com/commercetools/ui-kit/master/packages/components/inputs/radio-input/README.md

Reply:
Yeah, we need to fix all those READMEs.

---

Background: from the docs team I learned that the markdown and content structure is important and needs to follow certain rules. In this case, the main heading is H1, then the following headings should be H2, not H4. There is no point is skipping H2 and H3.

So we should do this:

```
#
##
###
####
```

instead of this:


```
#
####
```